### ISSUE TITLE
improved dmake (Trac #340)

### ISSUE THREAD

**Issue by danmar** [danmar/testissues#340]
Migrated from https://trac.cppcheck.net/ticket/340
```json
{
    "status": "closed", 
    "changetime": "2009-05-28T19:46:25", 
    "description": "Hi friends,\n\ni improved the tools/dmake.cpp about a feature:\n\nIt is now able build the project faster using multiple processes for compiling.\n\ncall the Makefile with: \"make fast_build [JOBS=2]\" \nThe default is two jobs in parallel. This is ideal for dual core machines. For example, on quad core machines you can set the number of JOBS to four and so on.\n\nI have attached the diff file\n\nReagards\n\nMartin\n", 
    "reporter": "ettlmartin", 
    "cc": "", 
    "resolution": "wontfix", 
    "_ts": "1243539985000000", 
    "component": "Other", 
    "summary": "improved dmake", 
    "priority": "", 
    "keywords": "", 
    "time": "2009-05-27T21:43:05", 
    "milestone": "", 
    "owner": "noone", 
    "type": "enhancement"
}
```
